docs/configuration/logging.md

```diff
@@ -1,44 +1,19 @@
 # Logging
 
-This page contains two different sections with different targets:
+There are different solutions for you to aggregate logs in your cluster for day to day operation.
 
-- Aggregate logs for day to day opperation
-- Collect logs [for support](#collect-logs-for-support)
+Overall, all solutions follow the same model: you have an **agent** on each node to collect and forward the logs, an **aggregator** to persist the logs, and a **UI** that allows you to analyze the logs.
 
-## Aggregation of logs
+One of the simplest stacks that you could use is:
```

Comment:
Its cool to have options but I still see no use case for the second. Actually if you use the second you have no way to get the logs to support